projects/media-viewer/src/assets/sass/bookmarks.scss

```diff
@@ -0,0 +1,44 @@
+.bookmark__save {
```

Comment:
can u wrap this in component tag - so that doesn't clash with other css 
i.e. 
component_name {
.bookmark__save